realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -655,6 +680,18 @@ class SyncedRealmTests {
                         assertEquals(expectedObjectIdList, it.columnObjectIdList)
                         assertEquals(expectedUUIDList, it.columnUUIDList)
                         assertEquals(expectedMixedList, it.columnMixedList)
+
+                        assertEquals(expectedObjectId, it.columnRealmDictionary["key"]!!.id)
+                        assertEquals(expectedString, it.columnStringDictionary["key"])
+                        assertTrue(Arrays.equals(expectedBinary, it.columnBinaryDictionary["key"]))
+                        assertEquals(expectedBoolean, it.columnBooleanDictionary["key"])
+                        assertEquals(expectedLong, it.columnLongDictionary["key"])
+                        assertEquals(expectedDouble, it.columnDoubleDictionary["key"])
+                        assertEquals(expectedDate, it.columnDateDictionary["key"])
+                        assertEquals(expectedDecimal128, it.columnDecimal128Dictionary["key"])
+                        assertEquals(expectedObjectId, it.columnObjectIdDictionary["key"])
+                        assertEquals(expectedUUID, it.columnUUIDDictionary["key"])
+                        assertEquals(expectedMixed, it.columnMixedDictionary["key"])
```

Comment:
Does this pass? Or did you just not get to it yet due to the missing cloud support. I think I saw a notice of the issues of dictionary with nullable links being fixed in the latest server image, so maybe worth a respin.

Reply:
Today's image doesn't contain the fix.